src/data/carriers/cheapCargoCarrierConfiguration.js

```diff
@@ -0,0 +1,11 @@
+import { AbstractCarrierConfiguration } from '@/data/carriers/abstractCarrierConfiguration';
+
+export class CheapCargoCarrierConfiguration extends AbstractCarrierConfiguration {
+  getDefaultConfig() {
+    return {};
```

Comment:
Heb je deze lege functie nodig? Is het misschien een optie om als defaultValue in `getDefaultConfig()` te zetten?